doc/hands_on/hands_on_dynamodb_vs.md

```diff
@@ -0,0 +1,256 @@
+   ```
+1. Create a schema to hold the adapter script:
+    ```sql
+   CREATE SCHEMA ADAPTER;
+   ```
+1. Create the Adapter Script:
+    ```sql
+    CREATE OR REPLACE JAVA ADAPTER SCRIPT ADAPTER.DYNAMODB_ADAPTER AS
+       %scriptclass com.exasol.adapter.RequestDispatcher;
+       %jar /buckets/bfsdefault/default/dynamodb-virtual-schemas-adapter-dist-0.4.0.jar;
+    /
+    ```
+1. Create UDF:
+    ```sql
+    CREATE OR REPLACE JAVA SET SCRIPT ADAPTER.IMPORT_DOCUMENT_DATA(
+      DOCUMENT_FETCHER VARCHAR(2000000),
+      REMOTE_TABLE_QUERY VARCHAR(2000000),
+      CONNECTION_NAME VARCHAR(500))
+      EMITS(...) AS
+        %scriptclass com.exasol.adapter.dynamodb.ImportDocumentData;
+        %jar /buckets/bfsdefault/default/dynamodb-virtual-schemas-adapter-dist-0.4.0.jar;
+    /
+   ```
+   
+## Create a Mapping Definition
+
+Now we need to tell the adapter how to map the DynamoDB documents to Exasol tables.
+For that, we create a file with the [Exasol Document Mapping Language](../gettingStartedWithSchemaMappingLanguage.md).
+
+You can create the file wherever you want. We will later upload it to the BucketFS. 
+
+`firstMapping.json`:
+```json
+{
+  "$schema": "../../main/resources/mappingLanguageSchema.json",
+  "srcTable": "Books",
+  "destTable": "BOOKS",
+  "description": "Mapping for the Books table",
+  "mapping": {
+    "fields": {
+      "Title": {
+        "toVarcharMapping": {
+          "varcharColumnSize": 254
+        }
+      }
+    }
+  }
+}
+``` 
+
+Now upload the mapping to BucketFS:
+
+```shell script
+curl -I -X PUT -T firstMapping.json http://w:writepw@<YOUR_DB_IP>:2580/default/mappings/firstMapping.json
+```
+
+## Create Virtual Schema
+
+Now we can create the Virtual Schema.
+
+Steps:
+
+1. Create a connection to DynamoDB
+    * For DynamoDB on AWS use:
+       ```sql
+      CREATE CONNECTION DYNAMO_CONNECTION
+         TO 'aws:<REGION>'
+         USER '<AWS_ACCESS_KEY_ID>'
+         IDENTIFIED BY '<AWS_SECRET_ACCESS_KEY>';
+      ```
+    * For a local DynamoDB use:
+       ```sql
+      CREATE CONNECTION DYNAMO_CONNECTION
+          TO 'http://<YOUR_DYNAMODB_IP>:8000'
+          USER 'fakeMyKeyId'
+          IDENTIFIED BY 'fakeSecretAccessKey';
+      ```
+      
+2. Create Virtual Schema:
+    ```sql
+   CREATE VIRTUAL SCHEMA DYNAMODB_TEST USING ADAPTER.DYNAMODB_ADAPTER WITH
+       CONNECTION_NAME = 'DYNAMO_CONNECTION'
+       SQL_DIALECT     = 'DYNAMO_DB'
+       MAPPING         = '/bfsdefault/default/mappings/firstMapping.json';
+   ```
+ 
+ ## First Results
+ 
+ Now you can browse the newly created table `DYNAMODB_TEST.BOOKS`:
+ 
+ ![alt text](result.png "Virtual Schema result")
+ 
+ You can see that the adapter mapped the title property of the documents to a column in the Exasol table.
+ 
+ ## Next Steps
+ In the next part of this series, we will show how to create more complex mappings.
+ 
```

Comment:
It's much better as an article now.  I would add some basic information about virtual schemas(couple od sentences) somewhere before you start creating an adapter and also emphasize that it's a new project and it brings something we haven't had before. Also, your article is quite long, I'm not sure how you are planning to post it, but theoretically, you could divide it into two articles. The first will contain information about DynamoDB, preparation stages, and some general description of your virtual schema. And the second one is a tutorial about creating virtual schema and mapping. I think you could also add more information about the mapping, it's a very interesting part, but it's also very short now.